### DISCUSSION TITLE
Launch application on macOS

### DISCUSSION THREAD

**Discussion by antonreshetov** [antonreshetov/timefall#19] [Announcements]
Since the application is not signed, the system will not allow it to be opened. However, it is possible to allow the launch of an unsigned application by performing the following actions:

- Run the application for the first time and close it
- Go to System Preference
- Click Security & Privacy
- Select the Allow button under the Allow apps downloaded from section

**Comment by patricklenert** [comment]
<img width="529" alt="image" src="https://github.com/antonreshetov/timefall/assets/1641188/6fb0646d-8d5f-4765-9eb9-405c0891d19c">

I have allowed all Apps on my Mac (M2 Max with Sonoma 14.4) but it says "App is damaged and can not be opened". On my windows PC it works fine.

**Reply by antonreshetov** [reply]
Try `sudo xattr -r -d com.apple.quarantine path/to/app`

**Comment by az1fr3** [comment]
I just want to say thank you, this app is exactly what i looking for. Simple, fast, relaible. KEEP THE GOOD WORK! PLEASE!!!

**Comment by kulterryan** [comment]
![Screenshot 2025-02-28 at 12 42 26 PM](https://github.com/user-attachments/assets/8e469b7c-18a3-43cb-a86c-1107f6889667)
